api/v1beta1/openstackdataplanerole_types.go

```diff
@@ -27,7 +27,7 @@ import (
 type OpenStackDataPlaneRoleSpec struct {
 	// +kubebuilder:validation:Optional
 	// DataPlaneNodes - List of nodes
-	DataPlaneNodes []DataPlaneNodeSection `json:"dataPlaneNodes,omitempty"`
+	DataPlaneNodes []OpenStackDataPlaneNodeSpec `json:"dataPlaneNodes,omitempty"`
```

Comment:
This is the heart of the patch: change the DataPlaneNodes list to type `OpenStackDataPlaneNodeSpec` instead of type `DataPlaneNodeSection`.

Reply:
looks good, `DataPlaneNodes` is still the type you're renaming, the CRDs are then regenerated accordingly.